sdk/servicelinker/azure-resourcemanager-servicelinker/src/samples/README.md

```diff
@@ -0,0 +1,57 @@
+<!-- ---
+page_type: sample
```

Comment:
any more metadata to help improve SEO, such as spring,  connect to db , service connector etc.

Reply:
Currently, No metadata is added, As it needs to change multiple script in multiple repo. We could discuss with PM and SDK Engineer System Team later.
Currently, I just commented them and make it not block Sample merge and release first.